packages/commons/src/auth/jwk.ts

```diff
@@ -64,3 +66,14 @@ export function sortJWK(jwk: JsonWebKey): JsonWebKey {
       {}
     );
 }
+
+export function getJwksClients(config: JWTConfig): JwksClient[] {
+  return config.wellKnownUrls.map((url) =>
+    jwksClient({
+      cache: true,
+      rateLimit: true,
+      cacheMaxAge: 60000, // 1 minute
```

Comment:
Why not put this value in the JWTConfig?

Reply:
@beetlecrunch  this commit [b3e378c](https://github.com/pagopa/interop-be-monorepo/pull/1002/commits/b3e378cf734c66e34a79743af8f519605b299744) add `jwksCacheMaxAge` configuration with default value like a [jwks-rsa](https://github.com/auth0/node-jwks-rsa/blob/master/EXAMPLES.md#configuration) library, otherwise it will override by provided value in env var `JWKS_CACHE_MAX_AGE`